content/blogposts/2024/cody-vscode-1-14-0-release.md

```diff
@@ -0,0 +1,99 @@
+--- 
+
+[Cody for VS Code](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai) v1.14.0 is now available! This update includes context window expansions, a new chat interface, easier ways to add context to chat, new models, and more.
+
+## Bigger and better context windows to improve chat
+
+We’ve historically limited Cody to a maximum context window of 7,000 tokens, meaning that Cody won’t pass more than 7,000 tokens (small chunks of text) to the underlying LLMs. This is mostly to make sure that _too much_ context isn’t passed into the LLM, which can lead to poor context recall and then poor answer quality. 
+
+Because of Claude 3’s [improved context recall](https://www.google.com/url?q=https://www.anthropic.com/news/claude-3-family%23:~:text%3DLong%2520context%2520and%2520near%252Dperfect%2520recall&sa=D&source=docs&ust=1713389874874139&usg=AOvVaw2xo3VN2mbBXrMP3p9cCkLc), we can now expand that context window without degrading chat quality. We’ve made huge updates to Cody’s maximum context window for Claude 3 Sonnet and Opus models:
+
+* 30,000 tokens of user-defined context (user text plus @-mentioned files)
+* 15,000 tokens of _continuous context_ (context that’s sent back to the LLM to help it recall earlier parts of a continuous conversation)
+
+_Note: One line of code converts to roughly 7.5 tokens, so these limits are approximately 4,000 and 2,000 lines of code, respectively._
+
+This update means two things:
+
+* You can now push _way more context_ into Cody, including multiple large files, so you can ask questions about larger amounts of code
+* You can have _much_ longer back-and-forth chats with Cody before it starts to forget context from earlier in the conversation
+
+We’ve also increased the _output_ token limit for all messages generated by Cody’s underlying LLMs. Outputs were previously limited to 1,000 tokens, and we’ve quadrupled it to 4,000 tokens. This means you shouldn’t see Cody's responses getting cut off mid-answer anymore. This update applies to all models and for all tiers of Cody users.
+
+## Revamped chat interface
+
+The chat interface is getting a new look! Cody’s chat log is now structured as a series of cells, each showing a message alongside the sender's avatar. When Sourcegraph fetches context to use in a request, Cody shows the context in a cell alongside the Sourcegraph avatar.
+
+<Figure
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1-14-release/new-chat-ui.png"
+  alt="Cody's new chat interface design"
+/>
+
+## Unit test command improvements for JS, TS, Go, and Python
+
+We’ve improved Cody’s unit test command for JavaScript, TypeScript, Go, and Python code in two ways:
+
+1. Cody will now determine the correct range of code to apply the command to based on the code syntax
+2. When you trigger a Quick Action (Ctrl + . / Cmd + .) on a function symbol, Cody will now given an option to generate a unit test
```

Comment:
Replace with `<kbd>Ctrl</kbd> + <kbd>.</kbd>`

Reply:
Resolved in new commit